src/BenchmarkDotNet/Environments/Runtimes/WasmRuntime.cs

```diff
@@ -26,18 +30,20 @@ public class WasmRuntime : Runtime, IEquatable<WasmRuntime>
         /// <param name="msBuildMoniker">moniker, default: "net5.0"</param>
         /// <param name="displayName">default: "Wasm"</param>
         /// <remarks>path to mainJs MUST be provided</remarks>
-        public WasmRuntime(FileInfo mainJs, string msBuildMoniker = "net5.0", string displayName = "Wasm", string javaScriptEngine = "v8", string javaScriptEngineArguments = "--expose_wasm") : base(RuntimeMoniker.Wasm, msBuildMoniker, displayName)
+        public WasmRuntime(FileInfo mainJs, string msBuildMoniker = "net5.0", string displayName = "Wasm", string javaScriptEngine = "v8", string javaScriptEngineArguments = "--expose_wasm", bool aot = false, DirectoryInfo runtimeSrcDir = null) : base(RuntimeMoniker.Wasm, msBuildMoniker, displayName)
         {
-            if (mainJs == null)
+            if (aot == false && mainJs == null)
```

Comment:
nit: style consistency with the repo

```suggestion
            if (!aot && mainJs == null)
```